package.json

```diff
@@ -41,6 +41,7 @@
     "start:embed-grid": "lerna run start --scope=@deephaven/embed-grid --stream",
     "pretest": "npm run build:necessary",
     "test": "jest --watch --changedSince origin/main",
+    "test:debug": "node --inspect-brk node_modules/.bin/jest --runInBand --watch --changedSince origin/main",
```

Comment:
I think I'd remove the `changedSince` flag for debug. I'd assume that we'll just specify the test we want when running this. You should be able to just do `npm run test:debug SomeFile`. Works without the double dash since it's not a flag you're passing

Reply:
done